src/main/java/ua/com/conductor/dao/GenericDao.java

```diff
@@ -0,0 +1,9 @@
+package ua.com.conductor.dao;
+
+import java.util.List;
+
+public interface GenericDao<T, I> {
```

Comment:
You have [functionality](https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_04.html) that is redundant at this state of project, I will list it below. So, for now, let's not use `Generic` interfaces. In fact, you won't be able to use them, or will be forced to change your corresponding interfaces